docs/LoaderDebugging.md

```diff
@@ -0,0 +1,312 @@
+[Layer Filtering](LoaderLayerInterface.md#layer-filtering) section of the
+[LoaderLayerInterface](LoaderLayerInterface.md) document.
+
+
+## Debugging Possible Driver Issues
+
+### Enable Driver Logging
+
+If you suspect a driver issue, set the loader logging to specifically output
+driver messages:
+
+```
+set VK_LOADER_DEBUG=error,warn,driver
+```
+
+Most important driver messages should go out with error or warning levels set,
+but this will provide more driver-specific info as well such as:
+  * What drivers are found
+  * Where they were found
+  * If there is any incompatibility with a given driver
+  * If any environment variables are disabling any of the drivers
+
+For example, the output of the loader looking for drivers on a Linux system may
+look like the following (NOTE: additional spaces have been removed from the
+output for easier reading):
+
+```
+DRIVER: Searching for driver manifest files
+DRIVER:    In following folders:
+DRIVER:       /home/$(USER)/.config/vulkan/icd.d
+DRIVER:       /etc/xdg/vulkan/icd.d
+DRIVER:       /etc/vulkan/icd.d
+DRIVER:       /home/$(USER)/.local/share/vulkan/icd.d
+DRIVER:       /home/$(USER)/.local/share/flatpak/exports/share/vulkan/icd.d
+DRIVER:       /var/lib/flatpak/exports/share/vulkan/icd.d
+DRIVER:       /usr/local/share/vulkan/icd.d
+DRIVER:       /usr/share/vulkan/icd.d
+DRIVER:    Found the following files:
+DRIVER:       /usr/share/vulkan/icd.d/intel_icd.x86_64.json
+DRIVER:       /usr/share/vulkan/icd.d/lvp_icd.x86_64.json
+DRIVER:       /usr/share/vulkan/icd.d/radeon_icd.x86_64.json
+DRIVER:       /usr/share/vulkan/icd.d/lvp_icd.i686.json
+DRIVER:       /usr/share/vulkan/icd.d/radeon_icd.i686.json
+DRIVER:       /usr/share/vulkan/icd.d/intel_icd.i686.json
+DRIVER:       /usr/share/vulkan/icd.d/nvidia_icd.json
+DRIVER: Found ICD manifest file /usr/share/vulkan/icd.d/intel_icd.x86_64.json, version "1.0.0"
+DRIVER: Found ICD manifest file /usr/share/vulkan/icd.d/lvp_icd.x86_64.json, version "1.0.0"
+DRIVER: Found ICD manifest file /usr/share/vulkan/icd.d/radeon_icd.x86_64.json, version "1.0.0"
+DRIVER: Found ICD manifest file /usr/share/vulkan/icd.d/lvp_icd.i686.json, version "1.0.0"
+DRIVER: Requested driver /usr/lib/libvulkan_lvp.so was wrong bit-type. Ignoring this JSON
+DRIVER: Found ICD manifest file /usr/share/vulkan/icd.d/radeon_icd.i686.json, version "1.0.0"
+DRIVER: Requested driver /usr/lib/libvulkan_radeon.so was wrong bit-type. Ignoring this JSON
+DRIVER: Found ICD manifest file /usr/share/vulkan/icd.d/intel_icd.i686.json, version "1.0.0"
+DRIVER: Requested driver /usr/lib/libvulkan_intel.so was wrong bit-type. Ignoring this JSON
+DRIVER: Found ICD manifest file /usr/share/vulkan/icd.d/nvidia_icd.json, version "1.0.0"
+```
+
+Then when the application selects the device to use, you will see the Vulkan
+device call chain reported in the following way (NOTE: additional spaces have
+been removed from the output for easier reading):
```

Comment:
I'm not sure what you mean, do you mean you want the raw output combined with above or you want all the spaces put back?  It seems like a waste of GitHub viewing space to have the spaces since text might require scrolling right to see.  When we could just remove the spaces...

Reply:
I think I meant that there was a `...` in the example output, and I felt that there shouldn't be an ellipsis instead all the output should be given. 
However, as I see the text now it looks fine, so I'm not sure what I was objecting too.